protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -190,17 +194,17 @@ Note that any other orders that the LP has on the book (limit orders, other pegg
-- If an LP order has offset set such that the resulting price falls outside of the price monitoring bounds the system adjusts it automatically so that it's placed on the previously violated bound (<a name="0038-OLIQ-009" href="#0038-OLIQ-009">0038-OLIQ-009</a>)
-- If the reference price itself is outside the valid price range the order should get placed at - when bid/ask is used as a reference, or one tick away from it - when mid is used as a reference. (<a name="0038-OLIQ-010" href="#0038-OLIQ-010">0038-OLIQ-010</a>)
+- [ ] Volume implied by the liquidity provision order is that given by [0034-PROB-liquidity_measure.feature](https://github.com/vegaprotocol/vega/blob/develop/integration/features/verified/0034-PROB-liquidity_measure.feature) in all the various scenarios there. (<a name="0038-OLIQ-001" href="#0038-OLIQ-001">0038-OLIQ-001</a>);
+- [ ] Volume implied by the liquidity provision order is that given by [0034-PROB-liquidity_measure.feature](https://github.com/vegaprotocol/vega/blob/develop/integration/features/verified/0034-PROB-liquidity_measure.feature) in all the various scenarios that test fractional order sizes (smallest order position of 0.01). (<a name="0038-OLIQ-002" href="#0038-OLIQ-002">0038-OLIQ-002</a>);
+- [ ] If an LP order has offset set such that the resulting price falls outside `[min_lp_vol_price, max_lp_vol_price]` then the system adjusts it automatically so that it's placed on the bound (<a name="0038-OLIQ-011" href="#0038-OLIQ-011">0038-OLIQ-011</a>)
 
 ### LP commitment order creation
-- A liquidity provisioning order must specify orders for both sides of the book (<a name="0038-OLIQ-003" href="#0038-OLIQ-003">0038-OLIQ-003</a>)
-- All orders created by an LP commitment must be pegged orders (<a name="0038-OLIQ-004" href="#0038-OLIQ-004">0038-OLIQ-004</a>)
-- Filled orders are replaced immediately to confirm to the LP commitment shapes (<a name="0038-OLIQ-005" href="#0038-OLIQ-005">0038-OLIQ-005</a>)
+- [ ] A liquidity provisioning order must specify orders for both sides of the book (<a name="0038-OLIQ-003" href="#0038-OLIQ-003">0038-OLIQ-003</a>)
+- [ ] All orders created by an LP commitment must be pegged orders (<a name="0038-OLIQ-004" href="#0038-OLIQ-004">0038-OLIQ-004</a>)
+- [ ] Filled orders are replaced immediately to confirm to the LP commitment shapes (<a name="0038-OLIQ-005" href="#0038-OLIQ-005">0038-OLIQ-005</a>)
+- [ ] Change of the market parameter `market.liquidity.volumeRange` which decreases the value will immediately tighten `[min_lp_vol_price, max_lp_vol_price]` and volume that was previously pegged inside the valid range and would now be outside is shifted to the bounds.   (<a name="0038-OLIQ-012" href="#0038-OLIQ-012">0038-OLIQ-012</a>)
```

Comment:
I'd go from immediately, "next time volumes" are recalculated, but I may be biased since I'm working on it. It's just that then we'd have to bake in a new trigger for re-deploying the orders. It's not a massive problem and I can definitely do it, but since we re-deploy those orders all the time anyway we might as well save ourselves some code complexity.

Reply:
Agree, modified accordingly.